From 25bec2e1064cfddd47174004bc9a08ad0381fd5b Mon Sep 17 00:00:00 2001
From: linmin <linmin@eswincomputing.com>
Date: Fri, 5 Jul 2024 18:07:00 +0800
Subject: [PATCH 099/109] WIN2030-15279:chore(main):Give a hint for production
 line test mode

Changelogs:
1.Print "***Production Line Test Mode!***\n" when MCU starts up if
  it's in production line test mode.

Signed-off-by: linmin <linmin@eswincomputing.com>
Change-Id: I730ac11231c90d059c78f73550d16d2ef618ee09
---
 Core/Src/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Core/Src/main.c b/Core/Src/main.c
index 7494186..061d416 100644
--- a/Core/Src/main.c
+++ b/Core/Src/main.c
@@ -159,6 +159,7 @@ void hf_main_task(void *argument)
   uart4_protocol_task_handle = osThreadNew(uart4_protocol_task, NULL, &protocol_task_attributes);
   daemon_keelive_task_handle = osThreadNew(deamon_keeplive_task, NULL, &daemon_keeplive_task_attributes);
   #if ES_PRODUCTION_LINE_TEST
+  printf("***Production Line Test Mode!***\n");
   protocol_task_handle = osThreadNew(protocol_task, NULL, &protocol_task_attributes);
   #else
   if (pdTRUE != xbspConsoleInit(CONSOLE_STACK_SIZE, CONSOLE_TASK_PRIORITY, &consoleHandle)) {
-- 
2.25.1

